packages/spectacle/package.json

```diff
@@ -37,15 +37,12 @@
   "dependencies": {
     "@types/testing-library__jest-dom": "^5.14.5",
     "broadcast-channel": "^3.2.0",
-    "broadcastchannel-polyfill": "^1.0.0",
-    "dedent": "^0.7.0",
```

Comment:
Nice, looks like we have our own `indentNormalizer` method that I'm assuming replaced this but we never got rid of the dep.